spec/models/ems_event_spec.rb

```diff
@@ -203,16 +203,25 @@
     let(:provider_event) { 'SomeSpecialProviderEvent' }
 
     it 'returns a list of groups' do
-      event_group_names = [
-        :addition, :application, :configuration, :console, :deletion, :general, :import_export, :migration, :network,
-        :power, :snapshot, :status, :storage
+      event_group_names = %i[
```

Comment:
it is :) `rubocop -a` changed that. Leave the code in a better shape than it was